### PR TITLE
update area

### PR DESCRIPTION
**Pull request by WangLiNaruto**

### CODE REVIEW

**Review by iskey — CHANGES_REQUESTED** [review]
Please also fix the examples in the Readme.

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/resource/Region.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import lombok.Data;
+
+/**
+ * The regions of the Cloud Service Provider.
+ */
+@Data
+public class Region {
+
+    private String name;
+    private String area;
```
Comment (swaroopar) [inline modules/models/src/main/java/org/eclipse/xpanse/modules/models/resource/Region.java:18]: can you also please add open API description and also validations for both fields

**Review by swaroopar — CHANGES_REQUESTED** [review]
OpenAPI details to be added.

**Comment by swaroopar** [comment]
Also what is the difference between this PR and #198

**Comment by iskey** [comment]
> Also what is the difference between this PR and #198

#198 is another way to implement the `area` thoughts.
It should be closed now for we decide to add the area as a field of `Region`.

**Review by swaroopar — APPROVED** [review]
looks good to me.

**Review by iskey — APPROVED** [review]
looks good.

**Pull request merged**